docs/chapters/stage1_inception.md

```diff
@@ -26,10 +26,10 @@ tags:
 #### a. decide the software programming language or framework for your research project.
 
  > *Narrative*: Explain that any programming language is fine to write the software which helps on the research of your scientific problem, but sometimes there are restrictions.
- > First of all, if you only have experience with a single programming language, then the choice of software code language used in the project is almost taken.
- > Any programming language (and its ecosystem) has its own strong points and weaknesses. Most of the time any programming language can be used. You should choose the one you are comfortable with.
- > But sometimes you do not have choice, because either you only know a single programming language, or you are aiming to use specialised hardware, and / or the existing libraries which will cut down the needed development time are tied to one or another language.
- > In some cases, it is the target ecosystem the determining point. Software communities like bioconductor, machine learning, bioconda or text mining, for instance, only provides feedback on their favoured programming languages.
+ > First of all, if you only have experience with a single programming language, then the decision regarding the software code language to be used in the project is almost taken.
```

Comment:
I'm not fully on board with this statement. It implies that the reader's knowledge is static and learning another programming language isn't a valid option. Some projects start with the intent of exploring the advantages of a new language or ecosystem. I would emphasize the ecosystem and community advantages of each language along with performance and time-to-code considerations.